README.md

```diff
@@ -30,6 +30,23 @@ Prophet is currently only supported via pip installation in EvalML.
 ```shell
 pip install evalml[prophet]
 ```
+Another option for installing Prophet with CmdStan as a backend is to use `make installdeps-prophet`.
+This command will do the following:
```

Comment:
As a beginner user, I'm a bit confused on how to use this command. This assumes that a user actually has the evalml repo; that is, I can't just create a new virtual env, pip install evalml, and then run this. I'd need to have cloned the evalml repo and run this while in the evalml repo, correct?

If that's true, we should make that clear! (though also makes me question how often people would take this route)